src/components/pages/Item/ExportAnalyticsPage.tsx

```diff
@@ -26,6 +31,16 @@ const CustomListItem = styled(ListItem)(({ theme }) => ({
 
 const ExportAnalyticsPage = (): JSX.Element => {
   const { t } = useAnalyticsTranslation();
+  const { itemId } = useParams();
+  const { data: memberships } = hooks.useItemMemberships(itemId);
+  const { data: currentMember } = hooks.useCurrentMember();
+
+  const userMemberships = memberships
+    ?.filter((m) => m.member.id === currentMember?.id)
+    .reduce((acc: PermissionLevel[], curr) => [...acc, curr.permission], []);
+
+  const memberPermissionOverItem =
+    userMemberships && PermissionLevelCompare.getHighest(userMemberships);
```

Comment:
Same as above.

Reply:
Also I think your implementation does not handle loading, what if the takes time to get the memberships/item? It will redirect directly no?